WooCommerce/WooCommerceTests/Notifications/LocalNotificationTests.swift

```diff
@@ -127,4 +127,21 @@ final class LocalNotificationTests: XCTestCase {
         assertEqual(expectedBody, notification.body)
         XCTAssertNil(notification.actions)
     }
+
+    func test_threeDaysAfterStillExploring_scenario_returns_correct_notification_contents() throws {
+        // Given
+        let scenario = LocalNotification.Scenario.threeDaysAfterStillExploring(siteID: 123)
+        let testName = "Miffy"
+        let stores = MockStoresManager(sessionManager: .makeForTesting(authenticated: true, displayName: testName))
```

Comment:
Nit: we probably don't need to mock store name for this test.

Reply:
Good catch. Removed it from other tests as well in https://github.com/woocommerce/woocommerce-ios/pull/10331/commits/a9b2db663c5f92d4d32b3dbf36da33a2695438a6